components/delegation/html/DelegationHTML.module.css

```
@@ -0,0 +1,99 @@
+.htmlContainer h1,
+.htmlContainer h2,
+.htmlContainer h3,
+.htmlContainer h4,
+.htmlContainer h5 {
+  float: none;
+  color: white;
+  font-family: "Montserrat", sans-serif !important;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Montserrat" (font-family-name-quotes)

(font-family-name-quotes)